safe/impact_function/multi_exposure_wrapper.py

```diff
@@ -1211,6 +1214,7 @@ def generate_report(
 
                 if exposure_groups:
                     for exposure_group in exposure_groups:
+                        analysis_layer = None
```

Comment:
Better to use the same wording as in the impact_function `impact_layer`? https://github.com/inasafe/inasafe/blob/develop/safe/impact_function/impact_function.py#L634